### PR TITLE
Make table and column autocomplete case insensitive

### LINKED ISSUE #24: [RFE] Make tab complete case insensitive
Hi Team, 

Using the tab complete function is fanatic but it's a small annoyance to have to use the correct case. 

If it is an easy change I think it would be worth it.

Example:
~~~
list logical_Flow <tab> <tab> Won't work. 
~~~

### PR DESCRIPTION
**Pull request by amorenoz**
This PR allows you to autocomplete even if you have started typing the name of the column or table in low case.

```
>>> list Logi <tab>
Logical_Router Logical_Switch
```
```
>>> list logi <tab>
logical_router logical_switch
```
```
>>> list logical_router N <tab>
Name Nat
```
```
>>> list logical_router n <tab>
name nat
```
```
>>> list logical_router nat <enter>
+----------------------+----------------------------------------+
|         NAME         |                  NAT                   |
+----------------------+----------------------------------------+
| GR_ovn-worker2       | [b504b2ba-fe7d-457c-8b23-48695e0aeda2] |
| GR_ovn-control-plane | [97558085-3fc9-4cf7-a1d2-66544b3be0d0] |
| GR_ovn-worker        | [305bf739-5dc7-408b-9d58-16b17641b8fc] |
| ovn_cluster_router   | []                                     |
+----------------------+----------------------------------------+
```

Fixes #24

### CODE REVIEW

File: cmd/ovsdb-mon/tables.go [hunk 1]
```diff
@@ -63,16 +63,16 @@ func NewStructPrinter(writer io.Writer, stype reflect.Type, fieldSel ...string)
 		for _, sel := range fieldSel {
 			found := false
 			for i := 0; i < stype.NumField(); i++ {
-				if stype.Field(i).Name == sel {
+				if stype.Field(i).Name == sel || strings.ToLower(stype.Field(i).Name) == sel {
```
Comment (flavio-fernandes) [inline cmd/ovsdb-mon/tables.go:66]: NIT: how about using EqualFold ?
```
if strings.EqualFold(stype.Field(i).Name, sel) {
```
https://golangbyexample.com/golang-case-insensitive-string-comparison/

File: cmd/ovsdb-mon/shell.go [hunk 2]
```diff
@@ -268,12 +274,29 @@ func (s *OvsdbShell) Run(ovsPtr *client.Client, args ...string) {
...
+					return exactTableFields[tableName]
+				}
+				return allTableFields[tableName]
 			},
 		}
 		listCmd.AddCmd(&subTableCmd)
 	}
+
+	// The list command autocompleter returns the exact table names if user has not
+	// started typing. If she has, then also include aliases (which are lower cased versions
```
Comment (flavio-fernandes) [inline cmd/ovsdb-mon/shell.go:288]: NIT: she/he  (be nice to the dudes too!) :)

**Review by flavio-fernandes — COMMENTED** [review]
Nice!!! I got 2 little nits. Otherwise, 
/lgtm

File: cmd/ovsdb-mon/tables.go (hunk 1 above)
Comment (amorenoz) [reply cmd/ovsdb-mon/tables.go:66]: Yes, we can use EqualFold. However, the autocomplete list will only have the ToLower version of the column name so unless the user goes wild with casing without the assistance of the autocomplete, it wouldn't actually make much difference.

File: cmd/ovsdb-mon/shell.go (hunk 2 above)
Comment (amorenoz) [reply cmd/ovsdb-mon/shell.go:288]: OK. I'll put both. I tend to default to "she" for inclusiveness (dudes are not underrepresented ;))

**Review by flavio-fernandes — APPROVED** [review]
nice!
/lgtm

**Comment by lewisdenny** [comment]
Nice! This seems to be working well with a quick test. I will let you know if I hit any issues, thanks a lot for implementing this!

**Pull request merged**